sdk/servicebus/service-bus/samples/v7/javascript/README.md

```diff
@@ -78,13 +78,13 @@ Take a look at our [API Documentation][apiref] for more information about the AP
 [session]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/session.js
 [scheduledmessages]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/scheduledMessages.js
 [useproxy]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/useProxy.js
-[advanced_administrationclient]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced/administrationClient.js
-[advanced_sessionroundrobin]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced/sessionRoundRobin.js
-[advanced_deferral]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced/deferral.js
-[advanced_listingentities]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced/listingEntities.js
-[advanced_sessionstate]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced/sessionState.js
-[advanced_movingmessagestodlq]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced/movingMessagesToDLQ.js
-[advanced_processmessagefromdlq]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced/processMessageFromDLQ.js
+[advanced_administrationclient]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced\administrationClient.js
+[advanced_sessionroundrobin]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced\sessionRoundRobin.js
+[advanced_deferral]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced\deferral.js
+[advanced_listingentities]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced\listingEntities.js
+[advanced_sessionstate]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced\sessionState.js
+[advanced_movingmessagestodlq]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced\movingMessagesToDLQ.js
+[advanced_processmessagefromdlq]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/samples/v7/javascript/advanced\processMessageFromDLQ.js
```

Comment:
wait, are we sure of this path change?

Reply:
That shouldn't have happened. I just ran the script "dev-tool samples publish --force". Though this shouldn't have happened. @willmtemple